src/test/java/com/uber/cadence/testUtils/TestServiceUtils.java

```diff
@@ -0,0 +1,100 @@
+/*
+ *  Copyright 2012-2016 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *  Modifications copyright (C) 2017 Uber Technologies, Inc.
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"). You may not
+ *  use this file except in compliance with the License. A copy of the License is
+ *  located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ *  or in the "license" file accompanying this file. This file is distributed on
+ *  an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *  express or implied. See the License for the specific language governing
+ *  permissions and limitations under the License.
+ */
+
+package com.uber.cadence.testUtils;
+
+import com.uber.cadence.*;
+import com.uber.cadence.internal.testservice.TestWorkflowService;
+
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.UUID;
+
+import static com.uber.cadence.internal.common.InternalUtils.createNormalTaskList;
+import static com.uber.cadence.internal.common.InternalUtils.createStickyTaskList;
+
+public class TestServiceUtils{
+    private TestServiceUtils() {}
+
+    public static void startWorkflowExecution(
+            String domain, String tasklistName, String workflowType, TestWorkflowService service)
+            throws Exception {
+        startWorkflowExecution(domain, tasklistName, workflowType, 100, 100, service);
+    }
+
+  public static void startWorkflowExecution(
+      String domain,
+      String tasklistName,
+      String workflowType,
+      int executionStartToCloseTimeoutSeconds,
+      int taskStartToCloseTimeoutSeconds,
+      TestWorkflowService service)
+      throws Exception {
+    StartWorkflowExecutionRequest request = new StartWorkflowExecutionRequest();
+    request.domain = domain;
+    request.workflowId = UUID.randomUUID().toString();
+    request.taskList = createNormalTaskList(tasklistName);
+    request.setExecutionStartToCloseTimeoutSeconds(executionStartToCloseTimeoutSeconds);
+    request.setTaskStartToCloseTimeoutSeconds(taskStartToCloseTimeoutSeconds);
+    WorkflowType type = new WorkflowType();
+    type.name = workflowType;
+    request.workflowType = type;
+    service.StartWorkflowExecution(request);
+    }
+
+    public static void respondDecisionTaskCompletedWithSticky(
```

Comment:
This overload does not seem to be very useful. IMO it's better to just keep the other version and use it.

Reply:
resolved.